src/NuGet.Jobs.Auxiliary2AzureSearch/README.md

```diff
@@ -0,0 +1,112 @@
+* The `SearchServiceName` setting is the name of your Azure Search resource. For example, use the name `foo-bar` for the Azure Search service with URL `https://foo-bar.search.windows.net`.
+* The `SearchServiceApiKey` setting is an admin key that has write permissions to the Azure Search resource.
+* The `AuxiliaryDataStorageContainer` and `StorageConnectionString` settings are the connection strings to your Azure Blob Storage account.
+
+```json
+{
+  "GalleryDb": {
+    "ConnectionString": "PLACEHOLDER"
+  },
+
+  "Auxiliary2AzureSearch": {
+    "AzureSearchBatchSize": 1000,
+    "MaxConcurrentBatches": 1,
+    "MaxConcurrentVersionListWriters": 32,
+    "SearchServiceName": "PLACEHOLDER",
+    "SearchServiceApiKey": "PLACEHOLDER",
+    "SearchIndexName": "search-000",
+    "HijackIndexName": "hijack-000",
+    "StorageConnectionString": "PLACEHOLDER",
+    "StorageContainer": "v3-azuresearch-000",
+    "StoragePath": "",
+    "AuxiliaryDataStorageConnectionString": "PLACEHOLDER",
+    "AuxiliaryDataStorageContainer": "ng-search-data",
+    "AuxiliaryDataStorageDownloadsPath": "downloads.v1.json",
+    "AuxiliaryDataStorageDownloadOverridesPath": "downloadOverrides.json",
+    "AuxiliaryDataStorageExcludedPackagesPath": "ExcludedPackages.v1.json",
+    "AuxiliaryDataStorageVerifiedPackagesPath": "verifiedPackages.json",
+    "MinPushPeriod": "00:00:10",
+    "MaxDownloadCountDecreases": 30000,
+    "EnablePopularityTransfers": true,
+    "Scoring": {
+      "PopularityTransfer": 0.99
+    }
+  }
+}
+```
+
+## Algorithm
+
+At a high-level, here's how Auxiliary2AzureSearch works:
+
+1. Update verified packages
+    1. Get the "old" list of verified package IDs from search auxiliary storage
+    2. Get the "new" list of verified package IDs from Gallery DB
+    3. Replace the verified package auxiliary file if needed
+1. Update downloads
+    1. Get the "old" downloads data from search auxiliary storage
+    1. Get the "new" downloads data from statistics auxiliary storage
+    1. Determine which packages have download changes
+    1. Get the "old" popularity transfers data from search auxiliary storage
+    1. Get the "new" popularity transfers data from statistics auxiliary storage
+    1. Determine which packages have popularity transfer changes
+    1. Update Azure Search documents in the "search" index to reflect the latest downloads and popularity transfers
+    1. Save the "new" downloads data to the search auxiliary storage
+    1. Save the "new" popularity transfers data to search auxiliary storage
+1. Update owners
+    1. Get the "old" owners data from search auxiliary storage
+    1. Get the "new" owners data from Gallery DB
+    1. Update Azure Search documents in the "search" index to reflect the ownership changes, if any
+    1. Track ownership changes in search auxiliary storage
+    1. Save the "new" owners data to the search auxiliary storage
```

Comment:
Bunch of ones here. Should be 1, 2, 3, etc.

Reply:
Yup see my comment here: https://github.com/NuGet/NuGet.Services.Metadata/pull/797#discussion_r449178372